README.md

```diff
@@ -32,6 +32,13 @@ behalf using the [GitHub
 OAuth](https://developer.github.com/v3/oauth/#web-application-flow) flow.
 Alternatively, users can comment on the GitHub issue directly.
 
+
+## Build requirements
+
+
+Python 3.10 is required to build the web pages.
+
+
```

Comment:
```suggestion

Python 3.10 is required to build the web pages.

```
We seem to just do a single blank like around headings in the rest of the file.

Reply:
Perhaps also explain why Python 3.10 is required so that we can know when to update the version.

Reply:
> Perhaps also explain why Python 3.10 is required so that we can know when to update the version.

Well, I did not did into _why_ it was required :) I tried 3.13, 3.12, and 3.11 locally. Then I checked what the CI used (3.10), and I tried that; success!

Reply:
This is the failure with Python > 3.10:

<details>

```console
Installing collected packages: Yapsy, sgmllib3k, PyRSS2Gen, zipp, watchdog, urllib3, unidecode, six, setuptools, ruamel.yaml.clib, Pygments, Pillow, piexif, natsort, multidict, MarkupSafe, markdown, lxml, idna, frozenlist, feedparser, docutils, cloudpickle, charset-normalizer, certifi, blinker, Babel, attrs, yarl, ruamel.yaml, requests, python-dateutil, mako, jinja2, importlib-metadata, aiosignal, doit, aiohttp, nikola
Successfully installed Babel-2.14.0 MarkupSafe-2.1.5 Pillow-10.2.0 PyRSS2Gen-1.1 Pygments-2.17.2 Yapsy-1.12.2 aiohttp-3.9.3 aiosignal-1.3.1 attrs-23.2.0 blinker-1.7.0 certifi-2024.2.2 charset-normalizer-3.3.2 cloudpickle-3.0.0 docutils-0.20.1 doit-0.36.0 feedparser-6.0.11 frozenlist-1.4.1 idna-3.6 importlib-metadata-7.0.2 jinja2-3.1.3 lxml-5.1.0 mako-1.3.2 markdown-3.2.2 multidict-6.0.5 natsort-8.4.0 nikola-8.2.2 piexif-1.1.3 python-dateutil-2.9.0.post0 requests-2.31.0 ruamel.yaml-0.18.6 ruamel.yaml.clib-0.2.8 setuptools-69.2.0 sgmllib3k-1.0.0 six-1.16.0 unidecode-1.3.8 urllib3-2.2.1 watchdog-4.0.0 yarl-1.9.4 zipp-3.18.1
Traceback (most recent call last):
  File "/Users/erlend.aasland/src/hpyproject.org.git/venv_nikola/lib/python3.12/site-packages/yapsy/PluginManager.py", line 132, in <module>
    import importlib.abc.Loader as imp
ModuleNotFoundError: No module named 'importlib.abc.Loader'; 'importlib.abc' is not a package

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/erlend.aasland/src/hpyproject.org.git/venv_nikola/bin/nikola", line 5, in <module>
    from nikola.__main__ import main
  File "/Users/erlend.aasland/src/hpyproject.org.git/venv_nikola/lib/python3.12/site-packages/nikola/__init__.py", line 39, in <module>
    from .nikola import Nikola  # NOQA
    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/erlend.aasland/src/hpyproject.org.git/venv_nikola/lib/python3.12/site-packages/nikola/nikola.py", line 49, in <module>
    from yapsy.PluginManager import PluginManager
  File "/Users/erlend.aasland/src/hpyproject.org.git/venv_nikola/lib/python3.12/site-packages/yapsy/PluginManager.py", line 134, in <module>
    import imp
ModuleNotFoundError: No module named 'imp'
make: *** [venv_nikola/bin/nikola] Error 1
```

</details>

Reply:
https://github.com/tibonihoo/yapsy/issues/19

Reply:
Thanks for digging!

Reply:
Further info: https://github.com/getnikola/nikola/issues/3719

Reply:
Upgrading to nikola 8.3.0 or newer should fix this issue.

Reply:
That's a good idea.